lib/utils/url_launcher.dart

```diff
@@ -1,8 +1,8 @@
 import 'package:url_launcher/url_launcher_string.dart';
 
-void launchURL(String url) async {
+Future<void> launchURL(String url) async {
   if (await canLaunchUrlString(url)) {
-    await launchUrlString(url);
+    await launchUrlString(url, mode: LaunchMode.externalApplication);
   } else {
     throw 'Could not launch $url';
   }
```

Comment:
Everything seems fine ! Check whether the url starts with mailto and take action on that.

Reply:
I have fixed the use of extra parameter's and function and made changes accordingly. Now mail and slack link on about page can open in mail app and browser respectively with just the url_launcher utils. please review the code and suggest if any other changes can be made.